ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml

```diff
@@ -4195,6 +4199,28 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-template-usertags
+- as: vsphere-ipi-vsphere7-f28
+  cron: 45 16 7 * *
+  steps:
+    cluster_profile: vsphere-elastic
+    env:
+      HOSTS: "2"
+      NETWORK_TYPE: nested-multi-tenant
+      VCENTER_VERSION: "7"
```

Comment:
@rvanderp3 Hi, I'm trying to add a vsphere 7 job here but looks like setting `VCENTER_VERSION: "7"` did not work, could you help to take a look? Thanks. Maybe we should also add vsphere 7 [here](https://github.com/openshift/release/blob/6d4d5769570d088a10367555cc9f68ab9ad9ae31/ci-operator/step-registry/ipi/conf/vsphere/nested/ipi-conf-vsphere-nested-commands.py#L72)
`sh-5.1$ govc about
FullName:     VMware vCenter Server 8.0.3 build-24322831
Name:         VMware vCenter Server
Vendor:       VMware, Inc.
Version:      8.0.3
Build:        24322831
OS type:      linux-x64
API type:     VirtualCenter
API version:  8.0.3.0
Product ID:   vpx
UUID:         f5058142-bd21-42e9-a5d2-d05066c2ca1d`